src/include/autoinstall/autoinst_dialogs.rb

```diff
@@ -82,13 +81,15 @@ def ProfileSourceDialog(original)
     # @return [String] device
     def DiskSelectionDialog
       Builtins.y2milestone("Selecting disk manually....")
-      tm = Storage.GetTargetMap
+      log.error("FIXME : Missing storage call")
```

Comment:
Use the `# storage-ng` comment so these missing parts can be easily found later. See e.g. https://github.com/yast/yast-installation/compare/storage-ng?expand=1